pytricia.c

```diff
@@ -79,10 +79,6 @@ _prefix_convert(int family, const char *addr, prefix_t* prefix) {
     int prefixlen = -1;
     char addrcopy[128];
 
-    if (strlen(addr) < 4) {
```

Comment:
I'd prefer to keep a check here for too-small address strings.  Clearly 4 is too long, but are there any valid addresses that can be represented with fewer than 2 characters?  My point is that it's better to just bail out if there's no point in even attempting to do anything else.